stardog/templates/monitoring/stardog-rules.yaml

```diff
@@ -24,13 +25,13 @@ spec:
         app: stardog
         severity: critical
     - alert: StardogLicenseExpire
-      expr: stardog_dbms_license_expiration_value < 42
+      expr: stardog_dbms_license_expiration_value{ {{ $ns_selector }} } < 42
       for: 10m
       annotations:
         summary: Stardog license about to expire
         description: The Stardog license will expire in {{ "{{" }} $value {{ "}}" }} days.
       labels:
         env: {{ .Release.Name }}
         app: stardog
-        severity: "{{ "{{ if $value lt 21 }}" }}critical{{ "{{ else }}" }}warning{{ "{{ end }}" }}"
+        severity: "{{ "{{ if lt $value 21.0 }}" }}critical{{ "{{ else }}" }}warning{{ "{{ end }}" }}"
```

Comment:
Do you know where I can find the documentation for operators in the severity field?

Reply:
https://prometheus.io/docs/prometheus/latest/configuration/template_examples/

I had to move the operator (`lt`) and use a float (`21.0`) for comparison.